docs/advanced/updates_github_actions.md

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Update lockfiles
         run: |
           set -o pipefail
-          pixi update --json | pixi exec pixi-diff-to-markdown >> diff.md
+          pixi update --json --no-install | pixi exec pixi-diff-to-markdown >> diff.md
```

Comment:
We removed it on purpose from the docs. It doesn't work if the manifest has pypi-dependencies

Reply:
ah okay, didn't notice that

Reply:
Yeah, I should have tagged you